src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" method="get" action="index.html">
+      <fieldset class="section">
+        <legend>
+          Personal informatiom:
+        </legend>
+        <label class="input">
+          Surname:
+          <input 
+            type="text" 
+            autocomplete="off" 
+            name="sureName"
+          >
+        </label> <br>
+        
+        <label class="input">
+          Name:
+          <input 
+            type="text"
+            autocomplete="off" 
+            name="name" 
+            required 
+          >
+        </label><br>
```

Comment:
```suggestion
        </label>
```
you can delete `<br>`, because it doesn't affect anything

Reply:
I don't fully understand. These elements are not blocks. And if I take away <br> everything will shift into lines.